### PR TITLE
Partial translation to French (2/3) of docs/index.md

### PR DESCRIPTION
**Pull request by lefessan**

### CODE REVIEW

File: site/docs/index.fr.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<!-- ((! set title Docs !)) ((! set documentation !)) -->
```
Comment (Chris00) [inline site/docs/index.fr.md]: I think you also want to `((! set nobreadcrumb !))`.  Landing pages are already clearly indicated by the header.

File: site/docs/index.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-<!-- ((! set title Docs !)) ((! set documentation !))
-((! set nobreadcrumb !)) -->
```
Comment (Chris00) [inline site/docs/index.md:2]: See above.

**Comment by lefessan** [comment]
I added the breadcrumbs because otherwise, the translation links are hidden behind the blocks: while still being visible, they cannot be clicked. I actually noticed the same problem appears on learn/index.html and learn/index.fr.html (at least, when using Chrome). The only fix I could find without modifying the CSS was to add the breadcrumbs.

**Comment by Chris00** [comment]
Ah OK.  Commit 378c09e should fix that.

**Comment by lefessan** [comment]
Ok, I added the breadcrumbs back

**Comment by Chris00** [comment]
Thanks.

**Pull request merged**